ruby-challenges.rb

```diff
@@ -1,16 +1,33 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
-# MINASWAN ✌️
+# MINASWAN ✌️ 
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+# Pseudocode: 1.) create a method called even_or_odd
+# 2.) even_or_odd will take in a number as an argument
+# 3.) use and if statment and an equality operator to make a evaluation
+# 4.) use % 2 == 0 to see if there is a remainder
+# 5.) return a string interpolated statement if the argument is even or odd
+
 reposts1 = 7
 # Expected output: '7 is odd'
 reposts2 = 42
 # Expected output: '42 is even'
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def even_or_odd repost
+    if (repost % 2 == 0)
+        "#{repost} is even"
+    else "#{repost} is odd"
+    end 
+end
+
```

Comment:
Nice use of the modulo! Make sure to keep your indentation and new lines uniform!